### PR TITLE
Qradar get scheme command

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28162

## Description
Added get-mapping-fields command

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.yml [hunk 1]
```diff
@@ -1300,6 +1300,10 @@ script:
       initial fetch state (will try to fetch first available offense).
     execution: false
     name: qradar-reset-last-run
+  - deprecated: false
+    description: Returns the list of fields for an incident type
```
Comment (Itay4) [inline Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.yml:1304]:
```suggestion
    description: Returns the list of fields for an incident type. This command should be used for debugging purposes.
```

File: Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py [hunk 2]
```diff
@@ -529,6 +530,26 @@ def upload_indicators_list_request(self, reference_name, indicators_list):
...
         )
 
+    def get_custom_fields(
+            self, limit: Optional[int] = None) -> List[dict]:
+        url = urljoin(self._server, "api/config/event_sources/custom_properties/regex_properties")
+        headers = self._auth_headers
+        fields = []
+        # Paging
+        start = 0
+        stop = 49
```
Comment (Itay4) [inline Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py:540]: why is that 49?

**Comment by Itay4** [comment]
please add to the readme how to use the schema mapping

File: Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py (hunk 2 above)
Comment (jochman) [reply Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py:540]: performance-wise. the API does not limit it but may take a long time to bring back lots of attributes.

**Comment by jochman** [comment]
https://github.com/demisto/content/pull/9142

**Pull request closed without merging**